cppapi/server/tango_config.h

```diff
@@ -230,6 +230,8 @@
 	#pragma warning(disable : 4355)
 	#pragma warning(disable : 4715)
     	#pragma warning(disable : 4786)
+    	#pragma warning(disable : 4267)
```

Comment:
Could you please add a comment here and explain what warnings are disabled by these 2 new lines, please?

Reply:
4267 --> var' : conversion from 'size_t' to 'type', possible loss of data --> lot of them
4244 --> conversion' conversion from 'type1' to 'type2', possible loss of data --> lot of them

Reply:
Sorry, my comment was not clear enough... I meant could you add comments in tango_config.h on these new #pragma warning(disable...) lines?